src/components/FullscreenModal/FullscreenModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { getColor, getDepth, getModalSize } from '../../utils/helpers';
+import { FlexContainer } from '../FlexContainer';
+import ModalContext from './Context/Context';
+import ModalHeader from './Header/Header';
+import ModalContent from './Content/Content';
+import ModalSidebar from './Sidebar/Sidebar';
+import ModalFooter from './Footer/Footer';
+import { FullscreenModalSizes } from './FullscreenModal.enums';
+import { FullscreenModalProps } from './FullscreenModal.types';
+
+export interface FullscreenModalSubcomponents {
```

Comment:
shouldn't this interface be in `FullscreenModal.types.ts` file?

Reply:
It would create dependency cycle when subcomponents are imported in `.types.ts`